app/services/importers/census/import.rb

```diff
@@ -0,0 +1,98 @@
+module Importers
+  module Census
+    class Import
+      SURVEY_ACS1 = 'acs1'
+      SURVEY_ACS5 = 'acs5'
+
+      COUNTIES_OKLAHOMA = 'Oklahoma'
+      COUNTIES_TULSA = 'Tulsa'
+
+      STATE_OKLAHOMA_FIPS = '40'
+
+      attr_accessor :variables, :survey, :year, :county_names, :state_fips, :zips, :statistics
+
+      def initialize(variables,
+                     survey,
+                     year,
+                     county_names: false,
+                     zips: false)
+        @variables = variables
+        @survey = survey
+        @year = year
+        @county_names = county_names
+        @state_fips = STATE_OKLAHOMA_FIPS
+        @zips = zips
+        @statistics = {}
+      end
+
+      def self.perform(*args)
+        new(*args).perform
+      end
+
+      def perform
+        import_variable
+        import_data
+      end
+
+      def import_variable
+        response = HTTParty.get(variable_url).as_json
+        survey = ::Census::Survey.find_or_create_by name: @survey, year: @year
+        variables.each do |variable|
+          statistics[variable] = ::Census::Statistic.find_or_create_by(
+            survey: survey,
+            name: variable,
+            label: response['variables'][variable]['label'],
+            concept: response['variables'][variable]['concept'],
+            group: response['variables'][variable]['group'],
+            predicate_type: response['variables'][variable]['predicateType']
+          )
+        end
+      end
```

Comment:
these methods are a bit long...(sorry I'm back in python land)
let me know if you want me to shorten